base/hiddenfile/hidden_file.go

```diff
@@ -1,6 +1,6 @@
 // +build !windows
 
-package base
+package hiddenfile
```

Comment:
how would you feel about merging both `linkfile` and this new `hiddenfile` into `fsi` instead? I feel like they're both part of our story around filesystem integration. Ended up needing to write a code spike to confirm no import cycles, here's a commit if you'd like to cherry-pick:

https://github.com/qri-io/qri/tree/hidden_and_link_file_to_fsi